package.json

```diff
@@ -123,17 +115,31 @@
       "sun.js"
     ]
   },
+  "devDependencies": {
+    "chai": "^4.3.4",
+    "chai-as-promised": "^7.1.1",
+    "cross-env": "^7.0.3",
+    "electron": "^13.1.2",
+    "electron-builder": "^22.11.7",
+    "husky": "^6.0.0",
+    "mocha": "^9.0.0",
+    "rimraf": "^3.0.2",
+    "sinon": "^11.1.1",
+    "spectron": "^15.0.0",
+    "standard": "^16.0.3"
+  },
   "dependencies": {
     "@electron/remote": "^1.1.0",
     "@meetfranz/electron-notification-state": "^1.0.0",
     "electron-log": "^4.3.5",
     "electron-store": "^8.0.0",
-    "i18next": "^20.2.2",
-    "i18next-node-fs-backend": "^2.1.3",
-    "luxon": "^1.26.0",
+    "i18next": "^20.3.1",
+    "i18next-fs-backend": "^1.1.1",
+    "luxon": "^1.27.0",
     "meeussunmoon": "^3.0.3",
+    "merge-img-vwv": "*",
```

Comment:
Is this needed in production build?

Reply:
In current implementation no. But if you want to combine short breaks circle then it would be useful.

Reply:
Ok, can you then remove the unused stuff? With new things it's hard for me to be sure :)

Reply:
removed.